### PR TITLE
MET-4990 Add external env environment

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +131,21 @@ We use jenkins to deploy. Make sure you can access [https://jenkins.eanadev.org/
...
+To make a (parameterised) docker image of the app first run these commands from the root of the project:
+
+`npm run build-shared`
+`npm run dist-metis`
+
+and then send the output of that command to a docker nginx image with this command:
+
+`docker build -t metis-ui-app-image:version projects/metis/`
+
+The generated docker image can then be ran with the command:
```
Comment (jeortizquan) [inline README.md:145]: add that the values of the variables in the `env_file` needed to be in place before building the image.

**Review by jeortizquan — COMMENTED** [review]
Just have a look on the suggestion comments, I ran it locally it worked.

File: projects/metis/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM nginx:1.18.0
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/metis/Dockerfile]: Changed.

File: projects/metis/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+CMD ["nginx", "-g", "daemon off;"]
+
+COPY ./src/static/nginx-docker.conf /etc/nginx/nginx.conf
+COPY ./src/static/nginx.types /etc/nginx
+
+COPY ./dist/metis /usr/share/nginx/html
+
+COPY ./env.copy.sh .
+RUN chmod 755 ./env.copy.sh
+ENTRYPOINT ["bash", "./env.copy.sh"]
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/metis/Dockerfile]: Changed.

File: README.md (hunk 1 above)
Comment (andyjmaclean) [reply README.md]: Changed.

Comment (jeortizquan) [reply README.md]: yes. as a new user took me a while to figure this out :joy: that is why I suggested it.

**Review by jeortizquan — APPROVED** [review]

**Pull request merged**